packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -163,16 +163,10 @@ class MultiSelectList extends React.Component {
       newProps.isSelected = isSelected;
     }
 
```

Comment:
This would also mean we don't need to check if the element is selectable in the click event.